sqlitex/pool_test.go

```diff
@@ -243,3 +247,58 @@ func TestPoolPutMatch(t *testing.T) {
 		dbpool1.Put(c)
 	}()
 }
+
+func TestPoolOpenInit(t *testing.T) {
+	t.Run("ok", func(t *testing.T) {
+		ctx := context.Background()
+		initScript := `
+                CREATE TABLE IF NOT EXISTS t(a INT, b INT);
+                CREATE TEMP VIEW v AS SELECT a FROM t;
+`
+		dbpool, err := sqlitex.OpenInit(ctx, poolURI, poolFlags, poolSize, initScript)
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func() {
+			if err := dbpool.Close(); err != nil {
+				t.Error(err)
+			}
+		}()
+
+		for i := 0; i < poolSize; i++ {
+			conn := dbpool.Get(nil)
+			defer dbpool.Put(conn)
+			if err := sqlitex.ExecScript(conn, `SELECT * FROM v;`); err != nil {
+				t.Fatalf("initScript not run on connection: %s", err)
+			}
+		}
+	})
+	t.Run("invalid initScript", func(t *testing.T) {
```

Comment:
The sub test names read well, but Go will just alter them anyway to fix characters that don't conform to its style. I think in this case you want `InvalidInitScript`.